yarn.lock

```diff
@@ -9883,11 +9883,16 @@ typedarray-to-buffer@^3.1.5:
   dependencies:
     is-typedarray "^1.0.0"
 
-typescript@^3.8.0-dev.20200111, typescript@^3.9.3:
+typescript@^3.8.0-dev.20200111:
```

Comment:
Mind running yarn-deduplicate so we end up with just _one_ TS 3.9.x?

Reply:
D'oh. Didn't mean to change this lockfile at all; this is residual from when I was manually testing things out with different TS versions. Will fix!